src/connectors/joyId.ts

```diff
@@ -48,7 +43,7 @@ export default class JoyIdConnector extends CKBConnector {
       return;
     }
     const ethAddress = await connect();
```

Comment:
After using `@joyid/ckb` to connect, should be rename `ethAddress` to `authData`.
see: https://docs.joyid.dev/guide/ckb/connect#get-started